### PR TITLE
add ability to parse cobertura files

### PR DESCRIPTION
**Pull request by ly-cultureiq**
This adds support for files that can be produced by pytest and coverage.py and many other job runners.

if this gets merged ill have a small PR for the github action to use this new setting

### CODE REVIEW

**Comment by nickmerwin** [comment]
Thank you @ly-cultureiq!

**Comment by XhmikosR** [comment]
I'm not sure this should have been merged @nickmerwin ...

cobertura-parse is archived and is also quite big. I mean for Christ's sake, they have mocha in dependencies...

IMO we should revert this and if you really want this to be added, we should fork the repo, bring it to a good shape and publish it on npm.

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
     "coverage",
     "coveralls"
   ],
-  "version": "3.0.7",
+  "version": "3.0.8",
```
Comment (XhmikosR) [inline package.json]: Also please never do this @ly-cultureiq. This is for the maintainer's to decide

Comment (ly-cultureiq) [reply package.json]: point taken - my apologies

**Pull request merged**